app/Views/Templates/components/elements/accordian.blade.php

```diff
@@ -0,0 +1,14 @@
+@props([
+    'title' => 'Accordion Title',  
```

Comment:
Similar to my feedback above the attributes should be aligned with the ones we have on our sheet